internal-client/src/main/java/org/apache/uniffle/client/impl/grpc/ShuffleServerGrpcNettyClient.java

```diff
@@ -154,7 +155,7 @@ public RssSendShuffleDataResponse sendShuffleData(RssSendShuffleDataRequest requ
             maxRetryAttempts,
             t -> !(t instanceof OutOfMemoryError));
       } catch (Throwable throwable) {
-        LOG.warn(throwable.getMessage());
+        LOG.warn(ExceptionUtils.getStackTrace(throwable));
```

Comment:
why not using `throwable` directly ?

Reply:
Because `LOG.warn` does not accept an exception type parameter, it only accepts a string.
As I mentioned above, I use `ExceptionUtils.getStackTrace(e)` just for the sake of minimal code changes and simplicity.
If you prefer using `throwable` directly, I will change it like below:
`LOG.warn("Failed to send shuffle data due to ", throwable);`

Reply:
`LOG.warn("Failed to send shuffle data due to ", throwable);` +1